main.py

```diff
@@ -86,7 +97,7 @@ def get_part_location(self, part_id):
         controller.colour_id = 5

```

Comment:
Make thee colour_id's take Enum values instead of magic numbers (to be more verbose). See: 
[https://docs.python.org/3/library/enum.html](https://docs.python.org/3/library/enum.html)

Eg:
class Item(Enum):
   Backcover = 0
   PCB = 1
  ...etc